cmake/pcl_find_cuda.cmake

```diff
@@ -31,7 +31,7 @@ if(CUDA_FOUND)
   # https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#deprecated-features
   
   if(NOT ${CUDA_VERSION_STRING} VERSION_LESS "11.0")
-    set(__cuda_arch_bin "3.5 3.7 5.0 5.2 5.3 6.0 6.1 6.2 7.0 7.2 7.5 8.0")
+    set(__cuda_arch_bin "3.5 3.7 5.0 5.2 5.3 6.0 6.1 6.2 7.0 7.2 7.5 8.0 8.6")
```

Comment:
This is technically wrong, as `8.6` was introduced with CUDA 11.1 and not 11.0. I think this will be solved with #4619 so it would be better to not touch it in this PR.

Reply:
Yeah. I've added a requires change comment regarding this above. This file shouldn't be in this PR.

Reply:
Ujuju, I'm sorry for this commit showing up here. I must have branched off from the wrong branch. Thanks for spotting this @SunBlack! @kunaltyagi is kindly teaching me how to use `git rebase`, and I will do that for this PR too!

Reply:
I managed to rebase the branch but do not know how to push them. After I have done what @kunaltyagi kindly outlined in #4719, the local branch is not tainted with any false commits anymore - Thank you very much for your help, @kunaltyagi ! 

I, however, do not understand how I can push the rebased local version. [SO's related question](https://stackoverflow.com/questions/19016698/git-branch-diverged-after-rebase) seems to indicate that I should do `git push -f`. Any `-f` is suspicious to me, so I wanted to make sure that this is correct indeed. How do I push the local branch after a rebase?

Reply:
It should be fine to force push the changes, and its required, as your local branch now has a new parent. Since you are the only working on this PR, a -f is fine, but otherwise -force-with-lease could be an option, see: https://stackoverflow.com/questions/52823692/git-push-force-with-lease-vs-force

Reply:
Thank you so much, @larshg ! Your detailed and instructive answers helped me a lot! I will try this tomorrow :-)

Reply:
Thank you again, Lars, for your kind help! I think the PR does not contain commits not intended for this PR anymore and the history seems to look fine too.